scripts/run_frontend_tests.sh

```diff
@@ -41,6 +41,19 @@ source $(dirname $0)/setup_gae.sh || exit 1
 # TODO(sll): Make this work with fewer third-party dependencies.
 bash scripts/install_third_party.sh
 
+#build so as to have minified js and css
+echo ""
+echo "  Runing build task with concatenation only "
+echo ""
+
+$NODE_PATH/bin/node $NODE_MODULE_DIR/gulp/bin/gulp.js build
+
+echo ""
+echo "  Runing build task with concatenation and minification"
+echo ""
+
+$NODE_PATH/bin/node $NODE_MODULE_DIR/gulp/bin/gulp.js build --minify=True
```

Comment:
So actually karma.conf.js receives args in array format in the form of 
```
[ 'node', 'command', 'path_to_karma.conf', ' <options>']
```
So it was unreasonable to add arg like ```minify=True```  for  it will be treated as  a single string. But gulpfile receives them as object, that is why it was easy to have something like ```argv.minify ``` which will give ```True``` as a value(minify being a key of argv).